File: pset6/mario/less/mario.py
from cs50 import get_int


while True:
    h = get_int("Height:")
    if h > 0 and h <= 8:
        break

# print_hash = "#"
# for i in range(1,h+1):
#     print((h-i)*" ", i*print_hash)

for i in range(h):
    for j in range(h):
        if i+j >= h-1:
            print("#", end="")
        else:
            print(" ", end="")
    print()

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

void get_positive_number(void);

int main(void)
{
    get_positive_number();

}

void get_positive_number(void)
{

    //prompting them for a positive integer between, say, 1 and 8, inclusive.
    int height;
    do


        height = get_int("Height:\n");

    while (height < 1 || height > 8);


    //# blocks building
    for (int i = 1; i <= height; i++)
    {


        for (int j = 1; j <= height - i; j++)
        {

            printf(" ");

        }
        for (int k = 1; k < i; k++)
        {
            printf("#");
        }

        printf("#\n");
    }

}